### ISSUE TITLE
`is_limit` label for some decays are not correct.

### ISSUE THREAD

**Issue by xijuUu** [particledatagroup/api#13]
Hi, I am trying to get the BF of `B+ --> J/psi(1S) K+`, but what I got is:
```
Decay: B+ --> J/psi(1S) K+
is_limit: True
0.001019886660276683
```
By comparing to the pdglive, apparently, it is not a limit value. I am using the lastest version of `pdg`, which is `0.1.1`. Could anyone please check this?

**Comment by juergberinger** [comment]
@xijuUu, thanks a lot for spotting this! This bug comes from wrong entries in the SQLite file. Will be fixed in the next update.

**Comment by juergberinger** [comment]
Fixed in version 0.1.2
